packages/ui/src/components/collectibles/native/collectible-text.native.tsx

```diff
@@ -30,15 +34,28 @@ export function CollectibleText({ src, size = 200 }: CollectibleTextProps) {
 
   if (!content) return null;
 
+  if (content.includes('<html')) {
```

Comment:
We get here if it's a text inscription but the text is actually `html`. I'm cautiously:
- removing any script tags
- disabling JS 
- disabling DOM storage

I did some testing and this will render inscribed HTML content correctly. 

Or other inscriptions classified as HTML we load the source directly in a WebView